packages/react/src/generators/remote/remote.ts

```diff
@@ -32,7 +32,6 @@ export async function remoteGenerator(host: Tree, schema: Schema) {
   const initApp = await applicationGenerator(host, {
     ...options,
     skipDefaultProject: true,
-    bundler: schema.bundler ?? 'webpack',
```

Comment:
Doesn't the applicationGenerator complain about missing `bundler`? We can just add `webpack` hardcoded there, if it complains?